skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -229,7 +240,7 @@ actual open class SkiaLayer internal constructor(
 
     @Volatile
     private var picture: PictureHolder? = null
-    private val pictureRecorder = PictureRecorder()
+    private var pictureRecorder = PictureRecorder()
```

Comment:
Agree with comment about symmetry. Either all init shall be in init function and var's be null inited, or, what I believe it better to move all changeable state to object with val's and dispose it as a whole.